### DISCUSSION TITLE
🚀 Windows系统Solo挖矿教程：3步开始挖CCX！

### DISCUSSION THREAD

**Discussion by nathanccxv** [CyberChainXyz/go-cyberchain#4] [Mining]
各位矿工朋友们好！👋 

想在Windows电脑上开始挖CCX吗？这份超简单的教程带你快速开始在CyberChain上进行solo挖矿！

## 🔧 第一步：添加CyberChain网络到MetaMask

1. 访问[CyberChain官网](https://cyberchain.xyz)
2. 点击"Add to MetaMask"按钮
3. 在MetaMask中切换到CyberChain网络
4. 复制你的MetaMask中的**EVM地址**

## 🖥️ 第二步：运行CyberChain节点

1. 从[go-cyberchain releases](https://github.com/CyberChainXyz/go-cyberchain/releases)下载`ccx-Windows-X64-v1.0.0-beta.zip`
2. 解压zip文件
3. 编辑`run_for_mining.bat`：
   - 将占位符(**0x123...fff**)替换为你的**EVM地址**
4. 运行`run_for_mining.bat`并等待同步

## ⛏️ 第三步：开始挖矿

1. 从[xMiner releases](https://github.com/CyberChainXyz/xMiner/releases)下载`xMiner-Windows-X64-v0.1.15-beta.zip`
2. 解压zip文件
3. 节点同步完成后，运行`solo_mining.bat`

搞定！你现在正在挖CCX了！🎉

### 💡 小贴士
- 确保你的GPU驱动是最新的
- 保持节点和挖矿软件同时运行
- 加入我们的社区获取支持和更新

祝挖矿愉快！🚀

**Comment by 3dpasspool** [comment]
Currently, it seems that Intel graphics cards are not supported. Can you update to support them

**Reply by nathanccxv** [reply]
重复问题，请参考：
https://github.com/orgs/CyberChainXyz/discussions/3#discussioncomment-11641536
